app/controllers/users_controller.rb

```diff
@@ -84,12 +84,17 @@ def user_params
     end
 
     def search_params
-      params.permit(:query)
+      params.permit(:query, :cityIds)
```

Comment:
використовуй рубішний неймінг
```suggestion
      params.permit(:query, :city_ids)
```

Reply:
ладн, назад поміняю, хах)

Reply:
resolved)